### ISSUE TITLE
How to show Accuracy and precision value at each epoch

### ISSUE THREAD

**Issue by ezekny** [alok-ai-lab/pyDeepInsight#22]
Hello, thanks for your support all the time. I have this issue again. How can i display the accuracy and precision value at every epoch during the training with CNN using Squeezenet. Thanks

**Comment by kaboroevich** [comment]
This issue is outside the scope of the package. There are plenty of examples on StackOverflow and similar sites on how to collect such information in a pyTorch training loop.
